gammapy/data/tests/test_event_list.py

```diff
@@ -19,6 +19,7 @@ def test_EventList():
     assert event_list.radec[0].to_string() == '82.7068 19.8186'
     assert event_list.galactic[0].to_string() == '185.956 -7.69277'
     assert event_list.altaz[0].to_string() == '46.2059 31.2001'
+    assert event_list.offset[0].value == 1.904497742652893
```

Comment:
This test fails:
https://travis-ci.org/gammapy/gammapy/jobs/104576296#L1505

For floating point number asserts, you have to use one of these assert statements:
https://gammapy.readthedocs.org/en/latest/development/howto.html#assert-convention